src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.Tests/LicenseScanTests.cs

```diff
@@ -67,6 +67,7 @@ public class LicenseScanTests : TestBase
         "cc-by-sa-4.0", // https://creativecommons.org/licenses/by-sa/4.0/legalcode
         "cc-pd", // https://creativecommons.org/publicdomain/mark/1.0/
         "cc-sa-1.0", // https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/cc-sa-1.0.LICENSE
+        "ecl-2.0", // https://opensource.org/license/ecl-2-0/
```

Comment:
This license isn't actually referenced in the source file. It should be classified as a false positive exclusion.